glommio/src/lib.rs

```diff
@@ -197,51 +211,51 @@
 //!
 //! ## Current limitations
 //!
-//! Due to our immediate needs which are a lot narrower, we make
-//! the following design assumptions:
+//! Due to our immediate needs which are a lot narrower, we make the following
+//! design assumptions:
 //!
-//!  - NVMe. Supports for any other storage type is not even considered.
-//!    This allow us to use io uring's poll ring for reads and writes which
-//!    are interrupt free. This also assumes that one is running either `XFS`
-//!    or `Ext4` (an assumption that Seastar also makes)
+//!  - NVMe. Supports for any other storage type is not even considered. This
```

Comment:
The need for NVMe is lifted. You can now use this with other types of media and it will work.
The design decisions are still assuming NVMe: it just won't refuse to work with other types of media.

So we could massage this phrase a bit.